functions.lua

```diff
@@ -229,6 +229,21 @@ function PacifistMod.remove_military_entities()
     end
 end
 
+function PacifistMod.disable_gun_slots()
+    local x_icon = "__core__/graphics/set-bar-slot.png"
+    local tool_icon = "__Pacifist__/graphics/slot-icon-tool.png"
+    local icon_kinds = { "gun", "ammo" }
+    for _, kind in pairs(icon_kinds) do
+        local icon = x_icon
```

Comment:
I love the idea. I am undecided whether we should just keep it simple and always use the tool icon

Reply:
I thought it would be confusing to have a slot that can never be used without an indication that it's disabled somehow.